packages/core/src/models.ts

```diff
@@ -289,11 +289,11 @@ export const models: Models = {
         },
         endpoint: "https://llm-gateway.heurist.xyz",
         model: {
-            [ModelClass.SMALL]: "meta-llama/llama-3-70b-instruct",
-            [ModelClass.MEDIUM]: "meta-llama/llama-3-70b-instruct",
-            [ModelClass.LARGE]: "meta-llama/llama-3.1-405b-instruct",
+            [ModelClass.SMALL]: settings.SMALL_HEURIST_LANGUAGE_MODEL || "meta-llama/llama-3-70b-instruct",
```

Comment:
```suggestion
            [ModelClass.SMALL]: settings.SMALL_HEURIST_MODEL || "meta-llama/llama-3-70b-instruct",
```
Probably update it to maintain consistency with the others?
https://github.com/ai16z/eliza/blob/main/.env.example#L73-L88

Reply:
The update sgtm!

Reply:
nice!